main.go

```diff
@@ -236,24 +249,27 @@ func addPodSecurityPrivilegedLabels(watchNamespaceName string) error {
 		setupLog.Error(err, "problem getting client")
 		return err
 	}
+	return addPodSecurityPrivilegedLabelsWithClientSet(watchNamespaceName, clientset)
+}
 
-	operatorNamespace, err := clientset.CoreV1().Namespaces().Get(context.TODO(), watchNamespaceName, metav1.GetOptions{})
+func addPodSecurityPrivilegedLabelsWithClientSet(watchNamespaceName string, clientset kubernetes.Interface) error {
+	nsPatch, err := json.Marshal(map[string]interface{}{
+		"metadata": map[string]interface{}{
+			"labels": map[string]string{
+				enforceLabel: privileged,
+				auditLabel:   privileged,
+				warnLabel:    privileged,
+			},
+		},
+	})
 	if err != nil {
-		setupLog.Error(err, "problem getting operator namespace")
+		setupLog.Error(err, "problem marshalling patches")
 		return err
 	}
-
-	privilegedLabels := map[string]string{
-		"pod-security.kubernetes.io/enforce": "privileged",
-		"pod-security.kubernetes.io/audit":   "privileged",
-		"pod-security.kubernetes.io/warn":    "privileged",
-	}
-
-	operatorNamespace.SetLabels(privilegedLabels)
-
-	_, err = clientset.CoreV1().Namespaces().Update(context.TODO(), operatorNamespace, metav1.UpdateOptions{})
+	fmt.Printf("Patches: %s\n", nsPatch)
+	_, err = clientset.CoreV1().Namespaces().Patch(context.TODO(), watchNamespaceName, types.StrategicMergePatchType, nsPatch, metav1.PatchOptions{})
```

Comment:
StrategicMergePatchType is naturally additive and should not remove existing labels as confirmed by the unit test.